WordPressKitTests/WordPressComRestApiTests.swift

```diff
@@ -60,6 +60,66 @@ class WordPressComRestApiTests: XCTestCase {
         }
     }
 
+    @available(iOS 16.0, *)
```

Comment:
Adding this availability declaration so that the `query(percentEncoded: false)` call compiles. However, it has no real effect on the unit tests run, because the pipeline runs on Xcode 15 using iOS 17 runtime.